.rubocop.yml

```diff
@@ -18,7 +18,7 @@ AllCops:
   - "**/Guardfile"
 Metrics/LineLength:
   Description: People have wide screens, use them.
-  Max: 200
+  Max: 260
```

Comment:
Had to increase the limit here because Rubocop was throwing errors in e.g. https://github.com/ekinanp/puppetlabs-cron_core/blob/MODULES-8306/lib/puppet/provider/cron/filetype.rb#L59 -- I would prefer not to break-up that message but I'm happy to update the error message if people object to this.